pkg/server/server.go

```diff
@@ -460,7 +461,7 @@ func (s *Server) HTTPHandler() (http.Handler, error) {
 
 	// List operator operands endpoint
 	operandsListHandler := &OperandsListHandler{
-		APIServerURL: s.KubeAPIServerURL,
+		APIServerURL: k8sProxyURL,
```

Comment:
<img width="726" alt="Screenshot 2024-02-27 at 14 21 00" src="https://github.com/openshift/console/assets/1668218/c4115d16-4968-4a5d-9a8e-75ce8489ba41">

Doing some detective work, I see this is my fault. Nice catch @TheRealJon 👍